docs/graphql.md

```diff
@@ -0,0 +1,68 @@
+# Using GraphQL
+
+\_Note: If testing this locally, you must have telescope running locally as well. Please refer to the documentation [here](./environment-setup.md)
+
+# Adding/Modifying Resolvers for GraphQL
+
+1. Our resolvers can be found in [here](../src/backend/web/graphql/index.js) under module.exports.resolvers.
+
+## Step-by-step Guide
+
+1. To access GraphQL, append /grpahql to the end of your local app URI. For example if your Telescope app is running at http://localhost:3000, you can access GraphQL for telescope at http://localhost:3000/graphql
+
+2. A list of available resolvers for Telescope can be found in src/backend/web/graphql/index.js under module.exports.resolvers. The list of available resolvers may change over time.
+
+3. We will use the getPosts query, this query has two parameters page and perPage. For example, if a user wanted to see the results of page 10 and have 30 results per page, he/she would type the following:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+
+  }
+}
+```
+
+This query won't run, because we also need to specify query fields. To see some available query fields type q within the getPosts() block:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+    q
+  }
+}
+```
+
+An available list of query fields will be displayed. Otherwise, please check the [definition](https://github.com/Seneca-CDOT/telescope/blob/08c55d4d694cf7b8a2c37b7945606c792976191d/src/backend/web/graphql/index.js#L15)
+
+Some of the available resolvers may not use any parameters in this case, please use resolvers like the following:
+
+```
+{
+  getPostsCount
+}
+```
+
+The following will let us retrieve the guid and author of all posts on the 10th page of results with 30(Posts) results shown:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+    query
+    author
+  }
+}
+```
+
+More than one query can be combined like the following:
+
+```
+{
+  getPostsCount
+    getPosts(page: 10, perPage: 30) {
+    query
+  author
+  }
+}
+```
```

Comment:
Same here
```
{
  getPostsCount getPosts(page: 10, perPage: 30){
    author
  }
}
```

Reply:
`query` is not a subfield included in our `Post`type so it can't be used there, but you're right that maybe @c3ho is using it there as placeholder. If that's the case, maybe it should be indicated in some way.